README.md

```diff
@@ -21,3 +28,22 @@ a full stack educational blog application written using three python web framewo
 6. related django tutorials:
    1. https://rasulkireev.com/managing-django-with-poetry/
    2. https://justdjango.com/blog/build-a-blog-with-django
+
+## contributing guidelines
+
+### before coding
+
+1. please contribute a design doc design doc for major features
+   1. a template can be found in `docs/template.md`
+   2. existing designs are listed in `docs/index.md`
+2. TODO: define a major feature
```

Comment:
Please define a major feature.

Reply:
1. alters db schema
2. if a change would result in more than 10% latency increase
3. any third party service integrations
4. any change involving more than 300 lines of code